lusidtools/cocoon/cocoon.py

```diff
@@ -98,7 +98,7 @@ def load_quote_batch(
             )
 
         return api_factory.build(lusid.api.QuotesApi).upsert_quotes(
-            scope=kwargs["scope"],
+            scope=kwargs["scope"], 
```

Comment:
What is changing here, looks like you're adding a whitespace?

Reply:
Ensuring black formatting happens